src/coffee/mapping.coffee

```diff
@@ -367,6 +367,8 @@ class Mapping
     if _.has langH2i, attribName
       _.each langH2i[attribName], (index, language) ->
         val = row[index]
+        if val == ""
```

Comment:
Any reason why you specifically checking for empty strings?

Better is you check for falsy values?